src/main/java/org/jitsi/videobridge/VideoChannel.java

```diff
@@ -237,6 +248,7 @@ private static synchronized RecurringRunnableExecutor getRecurringExecutor()
         throws Exception
     {
         super(content, id, channelBundleId, transportNamespace, initiator);
+        this.maxFrameHeight = MAX_FRAME_HEIGHT_DEFAULT;
```

Comment:
Why not initialize this with the field declaration? If we were to add a second constructor and forget to initialize it there it would default to 0.